js/web/web-backend.js

```diff
@@ -23,13 +21,11 @@ let workerPromise;
  */
 let numWorkers = undefined;
 
-let isInitialized = false;
+async function initializeBindings() {
+  wasm = plonkWasm();
```

Comment:
I moved this here to make the initial loading time even faster (< 20ms)

Reply:
Is there any use in returning from the function early if `wasm` is already defined? Let's say we accidentally call `initializeBindings()` twice, does that mean we have to wait for loading twice?

Reply:
Yeah that's very important! I moved the early return logic into snarky.web.js, through which this function will always be called